inventory/host_vars/_example.com/config.yml

```diff
@@ -27,8 +27,8 @@ mail_domain: example.com
 #custom_hba_entries:
 #  - { type: hostssl, database: "{{ db }}", user: zapier, address: '54.86.9.50/32', auth_method: md5 }
 #
-#custom_application_env_vars: |
-#  OFN_FEATURE_CONNECT_AND_LEARN: "true"
+#custom_application_dotenv_vars: |
+#  OFN_FEATURE_CONNECT_AND_LEARN="true"
```

Comment:
I prefer to keep the old name. It's accurately describing the content without mentioning the implementation. We may go with another gem at some point. :shrug:

Reply:
Hmmm... I think it's still safest to use separate variables during the transition (we can delete the old one later). How about renaming `custom_application_dotenv_vars` to just `custom_env_vars`?

Reply:
Yes, I like that. We don't need a reference to `application.yml` any more. :+1: